invenio_files_rest/tasks.py

```diff
@@ -62,7 +62,7 @@ def verify_checksum(
 
 def default_checksum_verification_files_query():
     """Return a query of valid FileInstances for checksum verification."""
-    return FileInstance.query
+    return FileInstance.query.filter_by(readable=True)
```

Comment:
compatibility: could you please use `db.session.query(FileInstance).filter_by(...)` instead of `FileInstance.query.filter_by`. I believe `db.session.query(FileInstance)` works with sqlalchemy <2.0 too, but `FileInstance.query` doesn't work anymore in sqlalchemy >= 2.0